### PR TITLE
Fix Tests for Importing non-csv Files

### PR DESCRIPTION
**Pull request by ctffarley**
* Change all instances of "seperator" to "separator"

There were multiple instances of this typo both in code and in comments which have now been fixed.

* Fix tsv tests

All of the tests for tsv files were pointing to the .csv file. In addition, that tsv file's content was a csv. The tests have been changed to use the tsv and the tsv is now a tab separated values file. An additional back slash was added to _IMPORT_EXTENSIONS to properly escape "\t" and not have it be parsed as a tab.

* Add test for custom delimiters

I added a test for a custom delimiter, in this case the pipe character "|" because that was not working as expected before.

* Fix overwriting the options dict

In admin.py there was a branch of code which is supposed append the options dict but instead would overwrite it. This will now produce the expected behavior.

* Fix inconsistent argument placement in docs and function

The constructor for ImportFile had separator as the 5th positional argument in the code but 3rd in the docs. Now it is 3rd to match the comment and ImportRaw constructor

* Import pytest in test_cache.py

This was added to make debugging tests in vscode work

### CODE REVIEW

File: stardog/content.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def __init__(
         Args:
           content (obj): Object representing the content (e.g., str, file)
           input_type (str): DELIMITED or JSON
-          seperator (str): Required if it's  DELIMITED CONTENT
+          separator (str): Required if it's  DELIMITED CONTENT
```
Comment (scardena) [inline stardog/content.py:173]: nit: there is an extra space in `it's  DELIMITED`

File: stardog/content.py [hunk 2]
```diff
@@ -218,9 +218,9 @@ def __init__(
         Args:
           file (str): Filename
           input_type (str): DELIMITED or JSON
-          separator (str): Required if it's  DELIMITED CONTENT
           content_type (str, optional): Content type
           content_encoding (str, optional): Content encoding
+          separator (str): Required if it's  DELIMITED CONTENT
```
Comment (scardena) [inline stardog/content.py:223]: nit: there is an extra space in it's  DELIMITED

**Review by scardena — CHANGES_REQUESTED** [review]
looks good, just a few nits above and one question:
In the description of the PR says: `Fix inconsistent argument placement in docs and function` but the function is not being modified. I think it was modified on a previous revision, but after the slack conversation maybe this was reverted, but the description of the PR was not. maybe update the description as well if this is true.

**Comment by ctffarley** [comment]
> looks good, just a few nits above and one question: In the description of the PR says: `Fix inconsistent argument placement in docs and function` but the function is not being modified. I think it was modified on a previous revision, but after the slack conversation maybe this was reverted, but the description of the PR was not. maybe update the description as well if this is true.

I rewrote the commit message to address this

**Review by scardena — APPROVED** [review]

**Pull request merged**